bloggies_backend/src/models/user.ts

```diff
@@ -59,9 +60,9 @@ export default class User {
   }
 
   /** Update the membership status after the application is complete and front end sends the status */
-  /** If status has been changed to "accepted", update membership start date and end date */
+  /** If status has been changed to "active", update membership start date and end date */
   static async updateMembership(user_id: number, appStatus: string) {
-    const now = appStatus === 'accepted' ? new Date() : null;
+    const now = appStatus === ACTIVE ? new Date() : null;
```

Comment:
This is a good change. Now, I think in the schema file StrawberryBasilFan should have an "active" status since that user has a membership start and end date.